src/config/dynamic-theme-fixes.config

```diff
@@ -4025,6 +4025,28 @@ IGNORE INLINE STYLE
 
 ================================
 
+www.realtek.com
+
+CSS
+#t3-header
+{
+    background: none !important;
+}
+#t3-header > div > div.col-xs-3.visible-xs-block > button.search-toggle
+{
```

Comment:
`{` on the same line as the selector only different selectors on seperate lines.

Reply:
Done.